INSTALL.md

```diff
@@ -47,6 +47,13 @@ variable, or it will take `~/.belenios` by default. Just run:
 On a modern desktop system, this needs approximately 20 minutes and 2.6
 gigabytes of disk space.
 
+If you intend to communicate the source files of belenios on your web server,
```

Comment:
This predication makes it sound like it is optional to communicate the sources of belenios, whereas it is mandated by the license (AGPL-3). I would remove it altogether.